packages/mobile/package.json

```diff
@@ -5,17 +5,15 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "start": "react-native start",
-    "start:bg": "react-native start &",
+    "start": "./scripts/run_app.sh",
```

Comment:
Just to clarify: `yarn start` now would fail with a proposal to specify a platform, correct?

Reply:
Correct. Maybe it's not worth it to have this still here?

Reply:
I think this is fine. I would assume, this is the first candidate to run when somebody has no experience with the codebase.